src/main/java/org/example/P05_TheOutputFor2.java

```diff
@@ -0,0 +1,11 @@
+package org.example;
+/* what is the output for System.out.println((2+3)4)*/
+public class P05_TheOutputFor2 {
```

Comment:
Rename to propper name. P02_OperationsWithBraketsPriority

Reply:
Class and branch renamed